src/shared/components/keep/StoreList.tsx

```
@@ -0,0 +1,147 @@
+                  exit={{ opacity: 0, y: -10 }}
+                  transition={{ duration: 0.2 }}
+                  className={`absolute right-0 top-[130%] rounded-2xl bg-white px-3 py-2 shadow-lg`}
+                >
+                  <ul className="whitespace-nowrap text-center text-sm font-medium">
+                    {groups?.map((g) => (
+                      <li
+                        className={`flex cursor-pointer items-center gap-1 border-b px-2 py-1 ${selectedGroup === g.name ? 'text-[var(--primary)]' : 'text-[var(--gray-5)]'}`}
+                        onClick={() => {
+                          handleGroupClick(g.name, g.storeCount, g.color, g.groupId);
+                        }}
+                      >
```

Biome (2.4.6)

[error] Missing key property for this element in iterable.

(lint/correctness/useJsxKeyInIterable)